UPGRADE-6.0.md

```diff
@@ -316,6 +316,9 @@ Security
    `UsernamePasswordFormAuthenticationListener`, `UsernamePasswordJsonAuthenticationListener` and `X509AuthenticationListener`
    from security-http, use the new authenticator system instead
  * Remove the Guard component, use the new authenticator system instead
+ * Deprecate methods `TokenInterface::isAuthenticated()` and `setAuthenticated`,
```

Comment:
Hmm this sentence sounds wrong to me, as they are already deprecated.

```
`TokenInterface:isAuthenticated()` and `setAuthenticated()` methods are removed without replacement
```
?

Reply:
"Remove .. and .. methods"

Reply:
👍 fixed